configure.ac

```diff
@@ -2514,7 +2514,7 @@ AS_IF([test x"$enable_pthread" = xyes], [
 	AC_CACHE_CHECK([arguments of pthread_setname_np], [rb_cv_func_pthread_setname_np_arguments],
 	    [rb_cv_func_pthread_setname_np_arguments=
 	    # Linux,AIX,  (pthread_self(), name)
-	    # NetBSD (pthread_self(), \"%s\", name)
+	    # NetBSD (pthread_self(), \"%s\", (void *)name)
```

Comment:
Is this cast necessary?
If so, the macro should contain it but not only in a comment.

Reply:
Yes I spotted as I got a warning about it.

Reply:
Re: https://man.netbsd.org/pthread_setname_np.3#SYNOPSIS,  the compiler doesn't know that `name` is a `printf` format string, and `arg` is not declared as `const`?

Reply:
ok let s drop this change that was nothing in fact.